taskt/Core/Automation/Commands/DataTable Commands/GetDataRowValueCommand.cs

```diff
@@ -1,67 +1,70 @@
+        [PropertyUISelectionOption("Index")]
+        [InputSpecification("Select whether the DataRow value should be found by index or column name.")]
+        [SampleUsage("")]
+        [Remarks("")]
         public string v_Option { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please enter the index of the DataRow Value")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter a valid DataRow index value")]
-        [Attributes.PropertyAttributes.SampleUsage("0 or [vIndex]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Search Value")]
+        [InputSpecification("Enter a valid DataRow index or column name.")]
+        [SampleUsage("0 || {vIndex} || Column1 || {vColumnName}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_DataValueIndex { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Assign to Variable")]
-        [Attributes.PropertyAttributes.InputSpecification("Select or provide a variable from the variable list")]
-        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required to pre-define your variables, however, it is highly recommended.")]
-        public string v_UserVariableName { get; set; }
+        [PropertyDescription("Output Value Variable")]
+        [InputSpecification("Select or provide a variable from the variable list.")]
+        [SampleUsage("vUserVariable")]
+        [Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required" +
+                 " to pre-define your variables; however, it is highly recommended.")]
+        public string v_OutputUserVariableName { get; set; }
 
         public GetDataRowValueCommand()
         {
-            this.CommandName = "GetDataRowValueCommand";
-            this.SelectionName = "Get DataRow Value";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
+            CommandName = "GetDataRowValueCommand";
+            SelectionName = "Get DataRow Value";
+            CommandEnabled = true;
+            CustomRendering = true;
             v_Option = "Index";
-
         }
 
         public override void RunCommand(object sender)
         {
-            var engine = (Core.Automation.Engine.AutomationEngineInstance)sender;
+            var engine = (AutomationEngineInstance)sender;
             var dataRowVariable = LookupVariable(engine);
             var variableList = engine.VariableList;
             DataRow dataRow;
```

Comment:
Changes required on lines 78 and 79:
- Remove extra line 78
- On Line 79, move Assignment Statement on next line to the else statement.

Reply:
done